packages/web/src/components/pages/Tests/TestDetails/TestSettings/SettingsVariables/Arguments.tsx

```diff
@@ -28,6 +28,10 @@ import {prettifyArguments} from '@utils/prettifyArguments';
 
 import {ArgumentsWrapper} from './Arguments.styled';
 
+const formatArgsArray = (args: string[]) => {
+  return args.map(arg => (arg.includes(' ') ? `"${arg}"` : arg));
```

Comment:
Missing escaping of `"` inside.

```
args:
- 'abc" def'
```
will become invalid `"abc" def"` instead of `"abc\" def"`

Reply:
Additionally, it sould escape quotes even when there is no space, otherwise:
```
args:
- 'abc"def'
```

will become `abc"def` instead of `abc\"def`

Reply:
Plus, it should check for any kind of space `/\s/.test(arg)`

Reply:
We could as well just use external library for quoting, like [**shell-quote**](https://www.npmjs.com/package/shell-quote)